client/fcm.js

```diff
@@ -0,0 +1,34 @@
+import firebase from "firebase/app";
```

Comment:
please do not import these here directly, please make them a dynamic import.

ie, 
```
import(/* webpackChunkName: "firebase-messaging" */ "./firebase-impl").then(({firebase}) => {
})
```